InstaScore/Controller/ChooseDateViewController.swift

```diff
@@ -65,21 +65,14 @@ class ChooseDateViewController: UIViewController, ScoreManagerDelegate{
 
     
     @IBAction func checkScorePressed(_ sender: UIButton) {
-        scoreManager.fetchScore(date1: date1, date2: date2)
-    }
-    
-    func didUpdateScore(scores:[ScoreModel]){
-        
-        
-        DispatchQueue.main.async {
-        let resultVC = ResultViewController()
-        resultVC.tescik = "HALKO"
-        }
-  
-//         print(scores[0].match_hometeam_name)
+       
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let resultVC =  storyboard.instantiateViewController(withIdentifier: "ResultStoryBoard") as! ResultViewController
```

Comment:
`as!` wiąże za sobą konsekwencje zcrashowania aplikacji jak jest użyte w niedobrym miejscu, dlatego swift umożliwia rozwiązywanie takich problemów przez tworzenie `guard let` albo `if let`

```suggestion
        guard let resultVC =  storyboard.instantiateViewController(withIdentifier: "ResultStoryBoard") as? ResultViewController else { return }
```


Dodatków `ResultStoryBoard` lepiej przerzucić jako statyczna zmienna w tym kontrolerze która przyjmuje tą wartość, jeżeli będziemy chcieli użyć go ponownie to wtedy korzystamy z tej statyczniej wartości a nie przypisujemy strona ponownie, więc eliminujemy możliwość pomyłki. Można też stworzyć structure ze statycznymi zmiennymi, która będzie odpowiedzialna za przypisywanie takich danych jak na przykład 
```swift
struct Constants {
    static let resultViewControllerID = "ResultStoryBoard"
}
```